README.md

```diff
@@ -1,6 +1,34 @@
-# osdfir-infrastructure
-This repository contains a set of Helm charts and deployment scripts that can be used to deploy Open Source Digital Forensics & Incident Response tools to a Kubernetes cluster. 
+# OSDFIR Infrastructure
+OSDFIR Infrastructure helps setup Open Source
+Digital Forensics tools to Kubernetes clusters using Helm. 
 
-This is still early stages of development and not usable yet, but please stay tuned for instructions on how to install our official release. 
+Currently, OSDFIR Infrastructure
+supports the deployment of the following tools:
+  * Timesketch; ref https://github.com/google/timesketch
+  * Turbinia; ref https://github.com/google/turbinia
 
-We look forward to sharing what is in store with the community soon!
+## TL;DR
```

Comment:
Won't this command as is fail because of missing required values/fields?  Should we have an example that has the minimum required fields set on the command line here?

Reply:
Per our chat updated this section to be more clear/point to gke installations doc